### ISSUE TITLE
Add a button to hide the synchronisation panel

### ISSUE THREAD

**Issue by laurent22** [laurent22/joplin#8611]
It should be a button with a down arrow. Clicking on it hides the panel, then the button becomes an "up" arrow. Clicking on it again, will show the panel.

It should be hidden by default.

![image](https://github.com/laurent22/joplin/assets/1285584/c2c15d9d-c145-4a5e-949b-af89cee1fad1)

**Comment by Abhishek-90** [comment]
I would like to take it up

**Comment by personalizedrefrigerator** [comment]
This seems to be related to [section 2.2.2](https://www.w3.org/TR/WCAG22/#pause-stop-hide) of the Web Content Accessibility Guidelines (see https://github.com/laurent22/joplin/issues/10795):
> **Moving, blinking, scrolling**
>    For any moving, blinking or scrolling information that (1) starts automatically, (2) lasts more than five seconds, and (3) is presented in parallel with other content, there is a mechanism for the user to [pause](https://www.w3.org/TR/WCAG22/#dfn-pause), stop, or hide it unless the movement, blinking, or scrolling is part of an activity where it is [essential](https://www.w3.org/TR/WCAG22/#dfn-essential); and

Currently, it is possible to hide the entire sidebar, however, this also hides the notebook and tag lists.

**Comment by Harsh-Bansal-13** [comment]
Hi I want to contribute to this issue. Since I am a new to open source contribution can anybody help me how to set up this project in local as I don't find anything related to guidlines for setting up in local

**Comment by mrjo118** [comment]
Is the propsoal here that only the summary of items synced is collapsible? Frankly the sync button itself and the timestamp of the last successful sync is very useful for quick access, and it would decrease usability if they were collapsed by default

**Comment by laurent22** [comment]
Yes maybe we should only hide the debugging information and leave timestamp and button.

**Comment by kanishka0411** [comment]
Hi, I'll take this one. Will add the up/down arrow toggle to collapse the debugging info while keeping the sync button and timestamp visible.

**Comment by kanishka0411** [comment]
Here is my current implementation of the sync panel with a ▼/▲ toggle arrow (collapsed and expanded).Does this direction look good?

<img width="1553" height="908" alt="Image" src="https://github.com/user-attachments/assets/aede8d3e-5c15-4006-b746-1b6c7073d7d0" />
<img width="1545" height="918" alt="Image" src="https://github.com/user-attachments/assets/6bbf6290-ad1c-4691-9535-ade037ed6fc6" />

**Comment by laurent22** [comment]
I think it should be like in the issue

**Comment by kanishka0411** [comment]
Updated it to match the issue mockup:-)

<img width="1146" height="914" alt="Image" src="https://github.com/user-attachments/assets/861346a0-aae2-4c40-8696-492ce94e69b5" />
<img width="1216" height="916" alt="Image" src="https://github.com/user-attachments/assets/bea1f643-89e0-4df0-9924-cfd0efed9ba6" />

**Comment by Ashutoshx7** [comment]
hy @kanishka0411  i saw you were working on this 
i actually made changes in my local was about to open the pr for this  then i read the conversation here
so i just wanted your permission should i move forward or not 
if you said no i would stop right there

**Comment by kanishka0411** [comment]
i am working on it and will open a pr soon

**Comment by Ashutoshx7** [comment]
> i am working on it and will open a pr soon

ya that's why i asked i am not opening my pr then 
best of luck


mockup looks good to me
